pkg/config/optimizely_config.go

```diff
@@ -28,6 +28,8 @@ type OptimizelyConfig struct {
 	FeaturesMap    map[string]OptimizelyFeature    `json:"featuresMap"`
 	SdkKey         string                          `json:"sdkKey,omitempty"`
 	EnvironmentKey string                          `json:"environmentKey,omitempty"`
+	AttributesMap  map[string]entities.Attribute   `json:"attributesMap"`
+	EventsMap      map[string]entities.Event       `json:"eventsMap"`
```

Comment:
Array. We have to exactly same names "Attributes" and "Events" defined in the TDD.

Reply:
Thanks! This will help fix the errors!